packages/maskbook/src/plugins/Polls/base.ts

```diff
@@ -0,0 +1,15 @@
+import type { Plugin } from '@masknet/plugin-infra'
+import { PLUGIN_ID, PLUGIN_ICON, PLUGIN_NAME, PLUGIN_DESCRIPTION } from './constants'
+
+export const base: Plugin.Shared.Definition = {
+    ID: PLUGIN_ID,
+    icon: PLUGIN_ICON,
+    name: { fallback: PLUGIN_NAME },
+    description: { fallback: PLUGIN_DESCRIPTION },
+    publisher: { name: { fallback: 'xichi' }, link: 'https://github.com/xichi' },
```

Comment:
cc @yisiliu or @Tedko

#1578

Reply:
```suggestion
    publisher: { name: { fallback: 'XiChi' }, link: 'https://github.com/xichi' },
```

Reply:
Patched in https://github.com/DimensionDev/Maskbook/pull/3500/commits/284d1c75db76106766b0e41b6f67cf2a3cae7115